### ISSUE TITLE
Build errors

### ISSUE THREAD

**Issue by acsbendi** [nytimes/gizmo#219]
When trying to build `simple_server.go` (as a dependency of another project), I get the following build errors:

```Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/simple_server.go:103:2: undefined: prometheus.InstrumentHandlerWithOpts
../gizmo/server/simple_server.go:123:3: undefined: prometheus.InstrumentHandler
../gizmo/server/simple_server.go:123:46: undefined: prometheus.UninstrumentedHandler
```

**Comment by jprobinson** [comment]
Howdy @acsbendi!

It looks like you may be attempting to build against the latest version of that prometheus client.

With Go Modules, [gizmo currently has that client pinned at version v0.9.2](https://github.com/nytimes/gizmo/blob/9eaa369806392cf61240a2bafe30df7c59f5b7c0/go.mod#L36), [where those do functions exist](https://github.com/prometheus/client_golang/blob/v0.9.2/prometheus/http.go#L181). 

Have you tried this using Go 1.11+ and Go Modules? Here's some documentation that should help: https://github.com/golang/go/wiki/Modules#how-to-install-and-activate-module-support

**Comment by acsbendi** [comment]
Yes, it's when I don't use Go modules. I just thought it's a good idea to give you a notice of the problem, since there might be a good reason why the missing parts were removed from prometheus. 

When I build with modules, everything is okay, thanks for the suggestion.

**Comment by rogeriozambon** [comment]
I'm running my apps with `dep` instead `go mod`. I just override the Prometheus' client to the pinned version by gizmo and worked well.

```toml
[[override]]
  name = "github.com/prometheus/client_golang"
  version = "0.9.2"
```

**Comment by alexellis** [comment]
Did we find a root cause for this? Is using an old version the only solution?
